### DISCUSSION TITLE
Begineer's guide to get a command line build working

### DISCUSSION THREAD

**Discussion by ort111** [Bodmer/TFT_eSPI#3096] [General]
Am using a PICO / PICO W for a project and am building other code fine / working / etc.  With this lib I don't seem to be able to build it the way that I think it should be.  Create a build directly got to it and cmake .. / then make .....etc...

 guide to help someone from scratch would be nice.  For a PICO do this, change this, pin it this way, etc.

I have browsed though the code and this is an impressive library.

Once can verify it is working, can then meld it into other projects, etc.

Display is an ILI9486 with Touch SPI and Display SPI.

Thanks

**Comment by Bodmer** [comment]
The library is intended for the Arduino build environment although some users have it working with the ESP IDF and with PlatformIO.
